bin/oref0-setup.sh

```diff
@@ -637,7 +637,7 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
         fi
         bluetoothdversioncompare=$(awk 'BEGIN{ print "'$bluetoothdversion'"<"'$bluetoothdminversion'" }')
         if [ "$bluetoothdversioncompare" -eq 1 ]; then
-            cd $HOME/src/ && wget -4 https://www.kernel.org/pub/linux/bluetooth/bluez-5.48.tar.gz && tar xvfz bluez-5.48.tar.gz || die "Couldn't download bluez"
+            cd $HOME/src/ && rm -rf bluez-5.48.tar.gz && wget -4 https://www.kernel.org/pub/linux/bluetooth/bluez-5.48.tar.gz && tar xvfz bluez-5.48.tar.gz || die "Couldn't download bluez"
```

Comment:
Does `rm -rf` return `true` if the file is not found?  May want to make `cd $HOME/src/ && rm -rf bluez-5.48.tar.gz` its own line before `cd $HOME/src/ && wget` etc. to make sure the `wget` doesn't incorrectly depend on the `rm`?

Reply:
I think both options are fine.

rm -rf should fail silently.

For example `rm -rf this-dir-does-not-exist && echo success` returns `success`
However `rm -rf this-dir-does-not-exist && echo success` returns `rm: cannot remove 'this-dir-does-not-exist': No such file or directory`

Reply:
If you specify the output file using the -O option it will overwrite any existing file.
Or we could use `curl http://server/folder/file1.html > file1.html`

Source: https://serverfault.com/questions/171369/how-to-allow-wget-to-overwrite-files or https://stackoverflow.com/questions/30418188/how-to-force-wget-to-overwrite-an-existing-file-
ignoring-timestamp

I think that's cleaner than removing a file that is non existing for most users.

Reply:
I agree overwriting is a cleaner option.

Reply:
I fixed this for golang (in the go-explorer-hat branch) by adding `-c` to wget, if you'd like to do the same for bluez.

Reply:
Updated with the `-c` flag. It's definitely a cleaner option. However I'm not familiar with the behaviour of this flag. To prevent silent failures like I experienced would it be better to use `-O` as suggested by @PieterGit?